src/LocationPicker.tsx

```diff
@@ -180,6 +182,26 @@ export default class LocationPicker extends Component<
       map.leafletElement.setView(startPort.center, startPort.zoom);
     }
   };
+  private renderTileLayer = () => {
+    const { geoserver, geoLayer, geoPort, geoWorkspace } = this.props;
+    const tileLayer = geoserver
+      ? {
+          url: `http://localhost:${geoPort}/geoserver/${geoWorkspace}/wms?`,
```

Comment:
I think it's better to allow the supplying of the entire url rather than the individual geoPort and geoWorkspace props. This simplifies the interface and is less restrictive; what if my geoserver isn't locally hosted?